src/store/actions/getLiqualityLiquidityForAsset.ts

```diff
@@ -0,0 +1,13 @@
+import { ActionContext } from '..';
+import { getSwapProvider } from '../../factory/swap';
+import { LiqualitySwapProvider } from '../../swaps/liquality/LiqualitySwapProvider';
+import { Network } from '../types';
+
+export const getLiqualityLiquidityForAsset = async (
```

Comment:
Hmm, Is this really an action? It doesn't mutate state, nor does it access any of the state context (`_context: ActionContext`)

This can be a utlity. In `src/swaps/utils.ts` instead